include/bluetooth/bluetooth.h

```diff
@@ -558,6 +572,34 @@ struct bt_le_adv_param {
 					     BT_GAP_ADV_FAST_INT_MIN_2, \
 					     BT_GAP_ADV_FAST_INT_MAX_2, NULL)
 
+/** Helper to declare periodic advertising parameters inline
+ *
+ *  @param _int_min     Minimum periodic advertising interval
+ *  @param _int_max     Maximum periodic advertising interval
+ *  @param _options     Periodic advertising properties bitfield.
+ */
+#define BT_LE_PER_ADV_PARAM_INIT(_int_min, _int_max, _options) \
+{ \
+	.interval_min = (_int_min), \
+	.interval_max = (_int_max), \
+	.options = (_options), \
+}
+
+/** Helper to declare periodic advertising parameters inline
+ *
+ *  @param _int_min     Minimum periodic advertising interval
+ *  @param _int_max     Maximum periodic advertising interval
+ *  @param _options     Periodic advertising properties bitfield.
+ */
+#define BT_LE_PER_ADV_PARAM(_int_min, _int_max, _options) \
+	((struct bt_le_per_adv_param[]) { \
+		BT_LE_PER_ADV_PARAM_INIT(_int_min, _int_max, _options)) \
+	})
+
+#define BT_LE_PER_ADV_DEFAULT BT_LE_PER_ADV_PARAM(BT_GAP_ADV_SLOW_INT_MIN, \
+						  BT_GAP_ADV_SLOW_INT_MAX, \
+						  BT_LE_ADV_OPT_NONE)
```

Comment:
Answer to: https://github.com/zephyrproject-rtos/zephyr/pull/25326#discussion_r446235302 
I think we should have a new options enum for periodic advertising options. I think that would avoid confusions as to which options can be used.

Reply:
Added its own enum

Reply:
Moved to new enum.